### DISCUSSION TITLE
exports is not defined in ES module scope

### DISCUSSION THREAD

**Discussion by algesthesiah** [vikejs/vike#1279] [Discussion] [answered]
Hi! Thanks for Reading

https://github.com/algesthesiah/vike-demo

**Comment by algesthesiah** [comment]
![image](https://github.com/vikejs/vike/assets/26083451/ed82234d-c0be-422b-931f-1ed3ad455630)

vike lastest examples react-full use commonjs、ssg vite build can find this bug

**Comment by algesthesiah** [comment]
https://github.com/algesthesiah/vike-demo

yarn build  show this bug

**Comment by brillout** [comment, marked as answer]
The reproduction is far from minmal. I'll only have a look at a reproduction that is minimal (Google is your friend to check what "minimal reproduction" means). Please respect the fact that OSS developers are very busy.
